static/app/views/performance/landing/index.tsx

```diff
@@ -212,6 +215,14 @@ export function PerformanceLanding(props: Props) {
                       router={props.router}
                       {...metricsDataSide}
                     />
+                    <DynamicSamplingAlert
+                      organization={organization}
+                      selectProject={
+                        selectedProjects.length === 1
+                          ? projects.find(project => project.slug === selectedProjects[0])
+                          : undefined
```

Comment:
selectedProjects are ids, not slugs

Reply:
correct. I just updated the code and will add tests :)